* mduft@gentoo.org: fix a wrong preprocessor directive (which is not noticed
                    on systems that have both "chown" and "utimes")
					breaks on Interix

http://bugzilla.gnome.org/show_bug.cgi?id=570069

--- glib-2.18.2.orig/gio/glocalfileinfo.c	2008-10-30 09:56:45 +0100
+++ glib-2.18.2/gio/glocalfileinfo.c	2008-10-30 09:58:04 +0100
@@ -2165,7 +2165,7 @@
 #ifdef HAVE_UTIMES
   GFileAttributeValue *mtime, *mtime_usec, *atime, *atime_usec;
 #endif
-#if defined (HAVE_CHOWN) && defined (HAVE_UTIMES)
+#if defined (HAVE_CHOWN) || defined (HAVE_UTIMES)
   GFileAttributeStatus status;
 #endif
   gboolean res;
